web/app/view/StateController.js

```diff
@@ -175,6 +176,14 @@ Ext.define('Traccar.view.StateController', {
         }
     },
 
+    selectEvent: function (position) {
```

Comment:
Why do we need a duplicate method?

Reply:
As always, a little hurried...
Now it has been tested more thoroughly and I think it’s ready for a review.